crates/adapters/src/integrated/delta_table/output.rs

```diff
@@ -493,7 +534,10 @@ mod test {
             input_file.write_all(b"\n").unwrap();
         }
 
-        let table_dir = TempDir::new().unwrap();
+        let mut storage_options = String::new();
+        for (key, val) in object_store_config.iter() {
+            storage_options += &format!("                {key}: \"{val}\"\n");
```

Comment:
why so many spaces?

Reply:
so it can get appended to yaml at the right nesting level.